typescript/package.json

```diff
@@ -1,49 +1,27 @@
 {
-  "name": "@coinbase/cdp-sdk",
-  "version": "1.1.2",
-  "description": "CDP SDK",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
-  "repository": "https://github.com/coinbase/cdp-sdk",
+  "private": true,
```

Comment:
This package.json is now the "root" package.json, i.e. the workspace manager. There is a new package.json at `src/package.json`, which contains the properties that are specific to the actual node module, and is shipped up to NPM with the package code.